client/dashboard/sites/settings-sftp-ssh/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useQuery } from '@tanstack/react-query';
+import { __experimentalVStack as VStack } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { file } from '@wordpress/icons';
+import { siteQuery, siteSftpUsersQuery, siteSshAccessStatusQuery } from '../../app/queries';
+import PageLayout from '../../components/page-layout';
+import { canUseSftp, canUseSsh } from '../../utils/site-features';
+import SettingsCallout from '../settings-callout';
+import SettingsPageHeader from '../settings-page-header';
+import calloutIllustrationUrl from './callout-illustration.svg';
+import EnableSftpCard from './enable-sftp-card';
+import SftpCard from './sftp-card';
+import SshCard from './ssh-card';
+
+export default function SftpSshSettings( { siteSlug }: { siteSlug: string } ) {
+	const { data: site } = useQuery( siteQuery( siteSlug ) );
+	const { data: sftpUsers } = useQuery( {
+		...siteSftpUsersQuery( siteSlug ),
+		enabled: site && canUseSftp( site ),
+	} );
+
+	const { data: sshAccessStatus } = useQuery( {
+		...siteSshAccessStatusQuery( siteSlug ),
+		enabled: site && canUseSsh( site ),
+	} );
+
+	const sftpEnabled = sftpUsers && sftpUsers.length > 0;
+
+	if ( ! site ) {
+		return null;
+	}
+
+	if ( ! canUseSftp( site ) ) {
+		return (
+			<PageLayout size="small" header={ <SettingsPageHeader title={ __( 'SFTP/SSH' ) } /> }>
+				<SettingsCallout
+					siteSlug={ siteSlug }
+					icon={ file }
+					image={ calloutIllustrationUrl }
+					title={ __( 'Direct access to your site’s files' ) }
```

Comment:
I think this was noted elsewhere, but the heading text is too small. It should be `Heading LG` which is `15px`.

<img width="640" alt="Screenshot 2025-06-03 at 11 40 54" src="https://github.com/user-attachments/assets/7b60f89b-7f06-492a-ad0b-0ed7bf669071" />